apps/web/src/components/forms/marker-create-edit-form.tsx

```
@@ -0,0 +1,404 @@
+
+          // Collections to add are those in the selected list but not in the current list
+          const collectionsToAdd = selectedCollections.filter(
+            (collectionId) => !inCollections!.includes(collectionId)
+          );
+
+          // Collections to remove are those in the current list but not in the selected list
+          const collectionsToRemove = inCollections!.filter(
+            (collectionId) => !selectedCollections.includes(collectionId)
+          );
+
+          delete data.collection_ids;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)